source/shared/core_stmt.cpp

```diff
@@ -2428,6 +2428,7 @@ void finalize_output_parameters( _Inout_ sqlsrv_stmt* stmt TSRMLS_DC )
     zend_string* key = NULL;
     void* output_param_temp = NULL;
 
+    try {
     ZEND_HASH_FOREACH_KEY_PTR( params_ht, index, key, output_param_temp ) {
```

Comment:
This block should now be indented.

Reply:
I originally did that but the changes are hard to spot, so I reverted and decided to make cosmetic changes later